src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/CreateSdkSymbolsLayout.cs

```diff
@@ -104,14 +104,14 @@ private IList<string> GenerateSymbolsLayout(Hashtable allPdbGuids)
 
                     if (guid != string.Empty)
                     {
-                        if (!allPdbGuids.ContainsKey(guid))
+                        if (!allPdbGuids.ContainsKey(GetKey(guid, file)))
                         {
                             filesWithoutPDBs.Add(file.Substring(SdkLayoutPath.Length + 1));
                         }
                         else
                         {
                             // Copy matching pdb to symbols path, preserving sdk binary's hierarchy
-                            string sourcePath = (string)allPdbGuids[guid]!;
+                            string sourcePath = (string)allPdbGuids[GetKey(guid, file)]!;
```

Comment:
This key was already computed from above. We should store that as a variable and reuse it here.

Same applies to the line further below: `allPdbGuids.Add(GetKey(guid, file), file);`

Reply:
Thanks - will fix.